### PR TITLE
Update Polish Translation

### PR DESCRIPTION
**Pull request by Gootector**
Update Polish Translation

Greetings,
Gootector

### CODE REVIEW

File: android/TuxGuitar-android/res/values-b+pl/strings.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
 	<string name="action_view_layout_show_chord_name">Pokaż nazwy akordów</string>
 	<string name="action_view_layout_show_chord_diagram">Pokaż diagramy akordów</string>
 
-	<string name="action_transport_play">Odtwarzaj/zapauzuj utwór</string>
+	<string name="action_transport_play">Odtwarzaj/wstrzymaj utwór</string>
```
Comment (guiv42) [inline android/TuxGuitar-android/res/values-b+pl/strings.xml:60]: @Gootector : please don't modify android `string.xml` file. These files are generated from the files in `common/resources` before release.
See here:
https://github.com/helge17/tuxguitar/blob/647c7826acbfa11ad0a551480192f52815b1b8a6/android/TuxGuitar-android/res/values-b%2Bpl/strings.xml#L5

**Review by guiv42 — CHANGES_REQUESTED** [review]
see my comment: android translation files shall not be modified manually

**Comment by Gootector** [comment]
> see my comment: android translation files shall not be modified manually

If it is as you typed here, it will be generated automatically. My changes will be replaced (or not) and I don't have to do anything with this PR. Please merge my PR. Thank you.

**Comment by guiv42** [comment]
Well, that's right, I could have merged it...
Thanks for the update

**Comment by Gootector** [comment]
> Well, that's right, I could have merged it... Thanks for the update

https://github.com/helge17/tuxguitar/commits/master/android/TuxGuitar-android/res/values-b%2Bpl - As you can see, the last changes (PR) before 1.6.4 release are made by me. No "generate from the files in common/resources before release"?

**Comment by guiv42** [comment]
Hmm, that's annoying.
The script to generate android translations from "common" files is [there](https://github.com/helge17/tuxguitar/blob/master/misc/messages_desktop_to_android.pl). The problem is that it's not automatized yet, so it's possible to forget this step before releasing.
I don't know what's best here. It seems difficult to me to ask all translators to run this perl script by themselves. We've got to think at a better solution.

**Comment by helge17** [comment]
The build script now checks if the Android translations are up to date when releasing a new stable version (8af678c32f238a1c4ad9e553cd03ca67cc5161b7).

**Pull request closed without merging**